program.py

```diff
@@ -39,7 +39,7 @@
 
 
 class EndpointTypes(enum.Enum):
-    OCS = 'OCS'
+    ADH = 'ADH'
```

Comment:
instead of doing this, could we have a warning that EndpointTypes.OCS is deprecated, and then somehow alias 'OCS' to 'ADH' so that existing code doesn't break? Not the end of the world if not, just might be better

Reply:
Translated OCS to ADH as in the temperature-sensor sample